src/com/sfuentes/Average.java

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Fortmat this file using this shortcut
⌥ (Option) + ⌘ (Command) + L 

And do the same to all other java files

Reply:
Already formatted